index.src.html

```diff
@@ -816,27 +831,29 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
   
   1. Set the <a>context object</a>'s <a>options</a> to |options|.
   
-  2. If it is not already present, add the <a>context object</a> to the 
-     list of <a>registered reporting observers</a> for the <a>environment
-     settings object</a> associated with the <a>context object</a>.
+  2. If it is not already registered, <a>register</a> the <a>context object</a>
+     with its associated <a>environment settings object</a>.
```

Comment:
why the asymmetry between "register" having a definition, but removing from the list being defined inline (i.e. why use a helper function for one but not the other)?  It seems simpler to me to stick with the original definition (no helper function) but that might be only because I haven't gotten to the part yet where this is helpful :-)

Reply:
I just thought that using the "registered" term made the many references to this property simpler and more clear, e.g. a step can be worded "If registered" instead of "If in the registered reporting observers list of the associated environment settings object". I then created the "register" helper to go with it, but I guess I'm only using it in one place, so I can remove it to fix the asymmetry.